robosats/settings.py

```diff
@@ -146,6 +146,9 @@
         }
     },
     "REDOC_DIST": "SIDECAR",
+    "ENUM_NAME_OVERRIDES": {
```

Comment:
Aha! TIL :smile:

Reply:
I also learned it today, well yesterday, while searching how to remove that warning ahahaha